### DISCUSSION TITLE
Demo/Walkthrough of the post-execute process.

### DISCUSSION THREAD

**Discussion by Halcinor** [frePPLe/frepple#495] [Q&A]
Hello all,

I have an installation configured following the production planning wizard. The wizard is complete and it is generating useful plans after execution. What I am now missing is the process of managing these now. Are there any walkthroughs, videos, guides, etc, that detail how to use and update fields like "Status" and "Receipt Date" and "End Date", etc. The help documentation hasn't proven fruitful in my searching as far as how these fields are specifically used and how they impact further production planning. 

A key example for me would be when I have a Routing operation, how do I update Frepple to represent the real world progress through the sub-steps such that I can execute revised plans and see timing slips, etc.

I feel like  this "after you complete the wizard" guide must be out there! Thanks for your help.

**Comment by jdetaeye** [comment]
The wizard is just a simplified quick start, nothing more and nothing less...   It is far from a complete implementation.

"After you complete the wizard" there are multiple follow-up tracks: 
- Explore the day-to-day common workflows and use cases: https://frepple.org/docs/current/videos/index.html
- Check out more advanced modeling constructs: https://frepple.org/docs/current/examples/index.html
- Import a more complete dataset from your ERP, most commonly done with a set of data extracts: https://frepple.org/docs/current/command-reference.html#importfromfolder

>A key example for me would be when I have a Routing operation, how do I update Frepple to represent the real world progress through the sub-steps such that I can execute revised plans and see timing slips, etc.

Frepple is a planning tool. It is not designed as the application where operators report the progress of the task. That remains the scope of your ERP system.  
From time to time (typically daily) data are extracted from the ERP system and a new plan is generated based on the latest sales orders, forecast, stock, production and purchasing status. 

After reviewing this plan, the planners will confirm/approve some manufacturing orders, purchase orders and distribution orders that are rolling within the "execution horizon" of the plan (which can range from a few days to a few weeks, depending on the industry).   These approved MO, DO and PO are then sent to the ERP system that will then created the required transactions and track their progress.

The part of the plan beyond this "execution horizon" remains in frepple. It is used to anticipate future bottlenecks and prepare appropriately.  This part of the plan is recomputed every time.
